lib/codegen/fromcto/vocabulary/vocabularyvisitor.js

```diff
@@ -16,6 +16,7 @@
 
 const VocabularyManager  = require('@accordproject/concerto-vocabulary').VocabularyManager;
 const util = require('util');
+const ModelUtil = require('../../../../../concerto-core/lib/modelutil.js');
```

Comment:
You might want to import it like this `const { ModelUtil } = require('@accordproject/concerto-core');`